multi-node.sh

```diff
@@ -11,6 +11,7 @@ libfabric_job_type=${libfabric_job_type:-"master"}
 # Current LibfabricCI IAM permissions do not allow placement group creation,
 # enable this after it is fixed.
 # export ENABLE_PLACEMENT_GROUP=1
+export USER_DATA_FILE=${JENKINS_HOME}/user_data_script.sh
```

Comment:
These scripts are also used by installer pipeline's test stage, where `$JENKINS_HOME` is not available.

Reply:
You can set this as a default value but can be overridden outside the scripts.

Reply:
I'll ping you directly